azure-pipelines/build.yml

```diff
@@ -43,6 +43,11 @@ steps:
     projects: '$(Build.SourcesDirectory)/src/Microsoft.Kusto.ServiceLayer'
     arguments: '--configuration Release'
 
+- task: UseDotNet@2
```

Comment:
maybe we should rewrite srgen in go so it's not framework-dependent anymore. Volunteers? :-)